ssl_test.go

```diff
@@ -645,6 +646,79 @@ func TestSslOpts(t *testing.T) {
 	}
 }
 
+func TestSslDialContext(t *testing.T) {
+	serverOpts := SslOpts{
```

Comment:
What does this test check?

Reply:
That dialContext works as regular dial (if we are not canceling it).

Reply:
So we could add the case with context to here, there is no need in a separate test case.
https://github.com/tarantool/go-tarantool/blob/master/ssl_test.go#L251

Reply:
Updated.